.github/workflows/android.yml

```diff
@@ -56,7 +56,7 @@ jobs:
         uses: bahmutov/npm-install@v1
         with:
           install-command: yarn --frozen-lockfile
-          persist-credentials: false
+          useLockFile: true
```

Comment:
why those keys are needed ? we have an example of npm-install usage in the current build flow (check in master)
why do not use it ?